### PR TITLE
first draft _ earning cred

### PR DESCRIPTION
**Pull request by benoxmo**
my first pull requests.... and largely drafted from @decentralion and @Bex2594 's post in discourse

### CODE REVIEW

File: docs/guides/earning-cred.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 ---
 title: 💧 Earning Cred
+description: How to get Cred from your contributions.
 ---
 
-TODO
+As soon as you interact with Sourcecred, you will be part of the Contribution Graph (i.e. a network of connected contributions) in which some [Cred] will flow or be minted in your direction. 
+
+
+Your contributions is tracked across the three active plugins we have for now. The types of contributions & plugins are likely to increase. But for now, small is beautiful.
```
Comment (Bex2594) [inline docs/guides/earning-cred.md:9]: Your contributions *are* tracked

File: docs/guides/earning-cred.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+
+
+| 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]|
```
Comment (Bex2594) [inline docs/guides/earning-cred.md:12]: Unfortunately the SourceCred emoji isn't showing :/

File: docs/guides/earning-cred.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+| -- | -- | -- |
+|someone gives your post : :sourcecred:  or :100: or :+1: emoji | someone gives your post/reply a :heart: | You open a pull request  |
+
+
+
+
+### 💧 Cred Minting
+the community decides to Mint some Cred for you, e.g. because they found something valuable. This Cred just magically appears out of nowhere in your direction. Say differently, you are now able to collect some drops of water from the sky
```
Comment (Bex2594) [inline docs/guides/earning-cred.md:20]: Love this analogy. I'd reword it slightly just to make it a bit clearer. Perhaps, "If we compare Cred to water, minting Cred would be like rain - it would fall from the sky, new water."

Comment (Bex2594) [inline docs/guides/earning-cred.md:20]: *The* community

File: docs/guides/earning-cred.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+
+
+
+### 🏄🏾Cred flowing
+Being part of the community, you share the same Contribution Graph. Your contribution is interconnected with others in all its different ways and across the time. Someone has responsed to your post. This same person has liked some other posts. Or later, another person will acknowledge your contribution. Cred will be flowing in/out on you but at a different speed.
```
Comment (Bex2594) [inline docs/guides/earning-cred.md:25]: Again, I really like the meaning behind this, and want to make it a bit clearer. How do you feel about this rewording?
"All the contributions you and your community make are on the same Contribution Graph. Each of your contributions is interconnected with other community contributions, even across different platforms. They even also through time - today's contributions connect to contributions from yesterday, last week, etc."

Comment (Bex2594) [inline docs/guides/earning-cred.md:25]: I'm not sure what you mean with "Someone has responsed to your post. This same person has liked some other posts. Or later, another person will acknowledge your contribution. Cred will be flowing in/out on you but at a different speed."

Could you maybe simplify it for now?

File: docs/guides/earning-cred.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+
+
+
+
+### 💧 Cred Minting
+the community decides to Mint some Cred for you, e.g. because they found something valuable. This Cred just magically appears out of nowhere in your direction. Say differently, you are now able to collect some drops of water from the sky
+
+
+
+### 🏄🏾Cred flowing
```
Comment (Bex2594) [inline docs/guides/earning-cred.md:24]: Comparing Cred to water has been a super helpful analogy for me personally in understanding and explaining Cred, which is why I'm so glad you're incorporating it into your explanation. Is there a way to incorporate more of the analogy into the explanation of flowing Cred, as you did for minting Cred?

File: docs/guides/earning-cred.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+Being part of the community, you share the same Contribution Graph. Your contribution is interconnected with others in all its different ways and across the time. Someone has responsed to your post. This same person has liked some other posts. Or later, another person will acknowledge your contribution. Cred will be flowing in/out on you but at a different speed. 
+
+Say differently, you are part of an environment that sustains itself across the time. This means you collect drops of water that will flow to someone's else direction downstream. Or some newcomers upperstream will provide you some fresh drops. But any point the sky&sun will make sure that some of your drops evaporate to keep everything in balance
```
Comment (Bex2594) [inline docs/guides/earning-cred.md:27]: This is a great recap, I especially love the evaporation part.

Comment (Bex2594) [inline docs/guides/earning-cred.md:27]: *To put it differently* would be a be a bit easier to read.
*across time*
*someone else's*
Maybe let's simplify the newcomers upstream part to "other contributors will provide you with their water." 'Newcomers' and 'fresh' could be confusing.

File: docs/guides/earning-cred.md [hunk 7]
```diff
@@ -1,5 +1,35 @@
...
+
+| 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]|
+| -- | -- | -- |
+|someone gives your post : [![](https://sourcecred.io/img/favicon.png)](https://sourcecred.io/img/favicon.png)  or :100: or :+1: emoji | someone gives your post/reply a :heart: | You open a pull request  |
+
+
+
+
+### 💧 Cred Minting
+The community decides to Mint some Cred for you, e.g. because they found something valuable. This Cred just magically appears out of nowhere in your direction. If we compare Cred to water, minting Cred would be like rain - it would fall from the sky, new water collected in your pond !
```
Comment (decentralion) [inline docs/guides/earning-cred.md:20]: I think it'd be helpful to explain how minting works in the three plugins right now:

- On GitHub, every pull request mints Cred
- On Discourse, every :heart: on a forum post mints Cred
- On Discord, certain reactions mint Cred

File: docs/guides/earning-cred.md [hunk 8]
```diff
@@ -1,5 +1,50 @@
 ---
 title: 💧 Earning Cred
+description: How to get Cred from your contributions.
```
Comment (Bex2594) [inline docs/guides/earning-cred.md:3]: "how to get Cred" makes it sound like there will be actionable items, maybe rephrase to "How you get Cred from your contributions"
